src/sass/partials/_bootstrap-custom.scss

```diff
@@ -5,40 +5,24 @@
  */
 
 // Core variables and mixins
-<<<<<<< HEAD:src/sass/partials/_bootstrap-custom.scss
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "bootstrap/variables";
 @import "bootstrap/mixins";
 
 // Reset and dependencies
 @import "bootstrap/normalize";
-=======
-@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables";
-@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins";
 
-// Reset and dependencies
-@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/normalize";
->>>>>>> 4d2d4413c73830ad76c174dffda6239fa646d6c1:src/sass/partials/_bootstrap.scss
 // @import "bootstrap/print";
```

Comment:
I meant all the lines commented out such as line 16 `// @import "bootstrap/print";`. If we aren't using them should just delete them